.github/workflows/typespec-validation.yaml

```diff
@@ -0,0 +1,25 @@
+name: Typespec Validation
+
+on: pull_request
+
+jobs:
+  typespec-validation:
+    name: Typespec Validation
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
```

Comment:
```suggestion
```

Reply:
Sorry to nitpick, but can we remove this `name`?  I think the default name of `actions/checkout@v4` is clear enough in the Action UI.